BricksAndHearts.UnitTests/ControllerTests/Landlord/LandLordControllerUnassignedTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿using BricksAndHearts.Controllers;
+using BricksAndHearts.Services;
+using BricksAndHearts.ViewModels;
+using Castle.Core.Internal;
+using FakeItEasy;
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Xunit;
+
+namespace BricksAndHearts.UnitTests.ControllerTests.Landlord;
+
+public class LandLordControllerUnassignedTests : LandlordControllerTestsBase
+{
+    private static AdminController _adminController = A.Fake<AdminController>();
+
+    [Fact]
+    public async void AddNewUnassignedLandlord()
```

Comment:
Would be good to have a test that non-admins cannot create unregistered landlords.